ingredient/views.py

```diff
@@ -1,3 +1,28 @@
-from django.shortcuts import render
+from django.core.exceptions import ObjectDoesNotExist
+from django.http            import JsonResponse
+from django.views           import View
+from recipe.models          import Recipe
+from ingredient.models      import Ingredient
 
-# Create your views here.
+
+class IngredientView(View):
+    def get (self,request,ingredient_id): 
+        try:
+            ingredient  = Ingredient.objects.get(id = ingredient_id) 
+            recipe_info = Recipe.objects.filter(ingredientrecipe__ingredient = ingredient_id) 
+            recipe_list = [{
+                "id"        : recipe.id,
+                "name"      : recipe.name,
+                "image_url" : recipe.image_url} for recipe in recipe_info] 
+            info_dict ={
+                "id"               : ingredient.id,
+                "name"             : ingredient.name,
+                "price"            : ingredient.price,
+                "info"             : ingredient.info,
+                "storage"          : ingredient.storage,
+                "image_url"        : ingredient.image_url,
+                "related_recipe"   : recipe_list
+            }
```

Comment:
아래와 같이 코드를 줄여볼 수 있지 않을까요?

```suggestion
            ingredient  = Ingredient.objects.get(id = ingredient_id)
            result = {
                "id"               : ingredient.id,
                "name"             : ingredient.name,
                "price"            : ingredient.price,
                "info"             : ingredient.info,
                "storage"          : ingredient.storage,
                "image_url"        : ingredient.image_url,
                "related_recipe"   : [{
                    "id"        : recipe.id,
                    "name"      : recipe.name,
                    "image_url" : recipe.image_url
                } for recipe in ingredient.recipe_set.all()] 
            }
```

Reply:
적용완료했습니다!  신세계~~